Packs/EDL/Integrations/EDL/EDL.yml

```diff
@@ -415,12 +415,26 @@ script:
       - 'False'
       - 'True'
     description: Updates values stored in the List (only available On-Demand).
-  dockerimage: demisto/flask-nginx:1.0.0.69390
+  - arguments: []
+    description: Returns the hosted content for the instance
+    name: get-edl
```

Comment:
Let's change the name to something indicating this is for test\debug and not for production purposes. How about get-edl-list-test ?